media-downloader/cmd/sync.go

```
@@ -0,0 +1,556 @@
+	erroredPayloads := downloadMedia(initialPayloads)
+
+	if len(erroredPayloads) > 0 {
+		logger.I.Warn("First download attempt completed with errors, starting retry...", zap.Int("errorCount", len(erroredPayloads)), zap.String("mediaType", mediaTypeFlag))
+
+		time.Sleep(5 * time.Second)
+		erroredPayloads = downloadMedia(erroredPayloads)
+
+		if len(erroredPayloads) > 0 {
+			errMsg := fmt.Sprintf("Some %s files could not be downloaded even after retrying.", mediaTypeFlag)
+			logger.I.Error(errMsg, zap.Any("payloads", erroredPayloads))
+			discord.SendLoggerWebhookMessage(baseClient, config.Env.DiscordWebhookUrl, discord.Error, errMsg, erroredPayloads)
```

golangci-lint (1.64.8)

Error return value of `discord.SendLoggerWebhookMessage` is not checked

(errcheck)